boost-common/src/main/java/boost/common/config/ConfigConstants.java

```diff
@@ -100,12 +100,19 @@
     public static final String JDBC_43 = "jdbc-4.3";
     public static final String JPA_21 = "jpa-2.1";
     public static final String JPA_22 = "jpa-2.2";
+    public static final String CDI_12 = "cdi-1.2";
     public static final String CDI_20 = "cdi-2.0";
     public static final String MPHEALTH_10 = "mpHealth-1.0";
+    public static final String MPOPENAPI_10 = "mpOpenAPI-1.0";
     public static final String MPRESTCLIENT_11 = "mpRestClient-1.1";
+    public static final String MPMETRICS_11 = "mpMetrics-1.1";
```

Comment:
Not your change at fault but it'd be nice if the constant reflected the fact that it was a Liberty feature.

Reply:
Given that this class keeps growing, it would be good to break this up. Perhaps having a "LibertyConfigConstants" class that lives in the Liberty runtime adapter or defining the constants in a more suitable class rather than a constants class. Maybe in the LibertyServerConfigGenerator class?  Or define them more granularly in the Liberty booster config classes themselves?

Reply:
> Not your change at fault but it'd be nice if the constant reflected the fact that it was a Liberty feature.

Given that this class keeps growing, it would be good to break this up. Perhaps having a "LibertyConfigConstants" class that lives in the Liberty runtime adapter or defining the constants in a more suitable class rather than a constants class. Maybe in the LibertyServerConfigGenerator class or define them more granularly in the Liberty booster config classes themselves. Either way, I think they should be pulled out of common since they are runtime specific. I can open an issue for this.